### ISSUE TITLE
Properly configure instance stores on different node types

### ISSUE THREAD

**Issue by jstjohn** [jtriley/StarCluster#203]
Starcluster works great, and properly mounts instance stores to `/mnt` when there is precisely 1 volume available. However whenever I mount to a node type with more than 1 volume, such as the `hi1.4xlarge` type which has 2 volumes that should be raided together, I need to do the raiding and everything myself. I suppose that I could do this manually, but it also seems like starcluster might be able to figure out which volumes are in `/dev` that aren't the EBS volumes that the user asks starcluster to mount, those can then be automounted.

**Comment by jtriley** [comment]
I think you mean the available ephemeral storage drives right? I think it should be feasible to do this while setting up /scratch (which is just symlinks to /mnt anyway). The code will need to be updated to check whether it has the necessary raid tools to do the job and skip the raid setup if not, otherwise probe and find all available ephemeral drives and raid them on /mnt. Perhaps in the case that the necessary raid tools are not found it just mounts them to /mnt, /mnt1, /mnt2, etc. as necessary?

**Comment by jstjohn** [comment]
Yes I am talking about the ephemeral storage drives. Basically every node type other than the ones with a single drive do not currently get mounted anywhere. And the ones with more than 4 drives you do not even see the remainder of the drives showing up in `/dev` (see my other issue related to the `hs1.8xlarge` instance). Mounting to `/mnt1 ... /mntN` would be reasonable. I think default when you boot an instance from the web console is `/media/ephemeral0` to `/media/ephemeralN` or something like that (maybe it is `/mnt/ephemeralN`, I don't remember exactly). Raiding everything together into a single drive on `/mnt` would really be the coolest option though.

**Comment by jtriley** [comment]
I think /mnt/ephemeralN in the case that raid tools are not available sounds reasonable, otherwise, we umount /mnt and raid all ephemeral drives there if necessary. This definitely wont make it before 0.94 release but I like this idea and plan to get to it unless someone else beats me to it.

**Comment by mortgoldman** [comment]
Did this make it into 0.94 release?  Doesn't seem to be working automagically for me...though that could easily be user error.

**Comment by jtriley** [comment]
@mortgoldman The latest 0.94.2 version of StarCluster enables all ephemeral drives for all instance types automagically but does not auto-mount them anywhere or set up a raid yet. Unfortunately there are other bigger issues lurking at the moment but I'm certainly open to a PR for this. I'm hoping to include this functionality eventually.
